docs/pages/releases/index.mdx

```diff
@@ -4,10 +4,14 @@ import { Status } from '../../components/primitives/Status';
 
 # Release Notes
 
-## 24th November 2021
+## 1st December 2021
 
 <Status look="latestRelease" />
 
+Keystone 6 has hit **General Availability**! We've shipped several more improvements since our release candidate last week. <Emoji symbol="🚀" alt="Rocket"/> — [Read more](/releases/2021-12-01)
```

Comment:
```suggestion
Keystone 6 has hit **General Availability**! We've shipped some additional improvements since our release candidate last week. <Emoji symbol="🚀" alt="Rocket"/> — [Read more](/releases/2021-12-01)
```